content/docs/1.1.1/deploy/upgrade/auto-uprade-engine.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Automatically Upgrading Longhorn Engine
+weight: 3
+---
+
+Since Longhorn v1.1.1, we provide an option to help you automatically upgrade Longhorn volumes to the new default engine version after upgrading Longhorn manager.
+This feature reduces the amount of manual work you have to do when upgrading Longhorn.
+There are a few concepts related to this feature as listed below:
+
+#### 1. Concurrent Automatic Engine Upgrade Per Node Limit Setting
+
+This is a setting that controls how Longhorn automatically upgrades volumes' engines to the new default engine image after upgrading Longhorn manager.
+The value of this setting specifies the maximum number of engines per node that are allowed to upgrade to the default engine image at the same time.
+If the value is 0, Longhorn will not automatically upgrade volumes' engines to the default version.
+The bigger this value is, the faster the engine upgrade process finishes.
+
+However, giving a bigger value for this setting will consume more CPU and memory of the node during the engine upgrade process.
+We recommend capping the value for this setting at `available_cpu/0.2` where `available_cpu` is the typical amount of idle CPU on the nodes in the cluster.
```

Comment:
I am not sure we should ask users to calculate like this.
1. `available_cpu/0.2` will be bigger than 3 for most environments.
2. The reason for choosing value 3 is more about leaving some room for error but don't overwhelm the system with too many failed upgrades.

Reply:
Okay. I removed the calculation and add the explanation for the value 3